locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -338,18 +334,18 @@ This philosophy can lead to some potentially problematic situations.
 Take this snippet for example:
 
 ```js
+let bar;
```

Comment:
`var` declaration is hoisted, with `let` declaration in the same place we have `ReferenceError: bar is not defined`